src/main/webapp/dev/allJsUnitTests.jsp

```diff
@@ -321,19 +335,17 @@
     <script type="text/javascript" src="/js/instructorFeedbackEdit/visibilityOptions.js" data-cover></script>
     <script type="text/javascript" src="/js/instructorFeedbackEdit/feedbackPath.js" data-cover></script>
     <script type="text/javascript" src="/js/instructorStudentList.js" data-cover></script>
-    <script type="text/javascript" src="/js/instructor.js" data-cover></script>
     <script type="text/javascript" src="/js/student.js" data-cover></script>
-    <script type="text/javascript" src="/js/index.js" data-cover></script>
 
-    <script type="text/javascript" src="/dev/CommonTestFunctions.js"></script>
-    <!-- Test scripts -->
-    <script type="text/javascript" src="/dev/CommonJsTest.js"></script>
-    <script type="text/javascript" src="/dev/InstructorCourseDetailsJsTest.js"></script>
-    <script type="text/javascript" src="/dev/InstructorEnrolmentJsTest.js"></script>
-    <script type="text/javascript" src="/dev/InstructorFeedbacksJsTest.js"></script>
-    <script type="text/javascript" src="/dev/InstructorStudentListJsTest.js"></script>
-    <script type="text/javascript" src="/dev/InstructorJsTest.js"></script>
-    <script type="text/javascript" src="/dev/StudentJsTest.js"></script>
-    <script type="text/javascript" src="/dev/SubmissionCountJsTest.js"></script>
+    <div id="test-scripts">
+        <script type="text/javascript" src="/dev/CommonJsTest.js"></script>
+        <script type="text/javascript" src="/dev/InstructorCourseDetailsJsTest.js"></script>
+        <script type="text/javascript" src="/dev/InstructorEnrolmentJsTest.js"></script>
+        <script type="text/javascript" src="/dev/InstructorFeedbacksJsTest.js"></script>
+        <!-- <script type="text/javascript" src="/dev/InstructorStudentListJsTest.js"></script> -->
```

Comment:
Yup; apparently the test was dependent on the `instructorStudentList.js`'s `document.ready` (unnecessary coupling!!), which has been stubbed now.